src/graphql/jest.config.js

```diff
@@ -1,5 +1,10 @@
-module.exports = Object.assign({}, require('../../jest.default.config'), {
-  "setupFiles": [
-    "./__tests__/__setup__/env-test.ts"
-  ],
-});
+module.exports = Object.assign(
```

Comment:
One of the reason why it is inside `src/graphql` is that jest's `--project` option allows as to run different test suites simultaneously with the same jest process. And in that case it reads config for every project located in project's root directory and named jest.config.js. Currently afaik there is no posibility to specify another config per project.
And one of the reasons why we use jest's project feature is that it allows generate accumulated coverage report for all projects when run like that.
So that is why I cannot see the way to put integration test's config into root for know. Any ideas?